src/core/event/index.js

```diff
@@ -296,13 +296,20 @@ export function Events(Base) {
     // =========================================================================
     /**
      * Handle rendering UI element updates and new content
+     * @param {string} name the site name
      * @void
      */
-    onRender() {
+    onRender(name) {
       const currentPath = this.router.toURL(this.router.getCurrentPath());
-      const currentTitle = dom.find(
-        `.sidebar a[href='${currentPath}']`,
-      )?.innerText;
+      const currentSection = dom
+        .find(`.sidebar a[href='${currentPath}']`)
+        ?.getAttribute('title');
+
+      const currentTitle = name
+        ? currentSection
+          ? `${name} - ${currentSection}`
```

Comment:
I suggest putting the site title after the page title.

```suggestion
          ? `${currentSection} - ${name}`
```

Reply:
I think the format are both fine, and I test them locally.
Personally I agree with @trusktr  to do something like `docsify - Configuration` to mention the site first.
Cos I checked some of the v4 sites behaviors such as the [docsify-themeable/QuickStart](https://jhildenbiddle.github.io/docsify-themeable/#/quick-start?id=new-site), they looks fine.

Maybe we could have a discuss on this.

Reply:
Also, we could introduce a `pageTitleFormatter` config, by default use the `name - title`, users can concat them on what they want in v5+.
The Formatter kinda like:
```js
<script>
  window.$docsify = {
    pageTitleFormatter: (config, section) => { },
  
  };
</script>
```

Reply:
I would also prefer page title first, then site title, as page name is often the more valuable source of contextual info (i.e. looking at open tab titles, list of history, bookmarks etc.).